### DISCUSSION TITLE
Lagunages in React-storefront

### DISCUSSION THREAD

**Discussion by pasith3e** [saleor/saleor#9434] [Q&A] [answered]
Hello,

I´m currently creating a Store with the new React-storefront. 
And I have so Problems with Create a Version of the Storefront in Germany  .
I added Germany to the Language Selection Menu:
region.ts:
```
export const LOCALES = [
  {
    slug: "de-DE",
    code: "DE_DE" as LanguageCodeEnum,
    name: "Deutsch",
  },
  {
    slug: "en-US",
    code: "EN_US" as LanguageCodeEnum,
    name: "American English",
  }
];
```

But when I change a value Inside of locale/de-DE.json 
`  "app.product.variantNotChosen": "Bitte wähle eine Variante",`
I still get the old text on the Storefront.
Is there anything else I have to change for it to work?

Thanks for you and best regards,
Jonas

**Comment by krzysztofwolski** [comment, marked as answer]
Hello!
Is the JSON file with the translations imported?
https://github.com/saleor/react-storefront/blob/main/components/RegionsProvider/RegionsProvider.tsx#L32

**Reply by pasith3e** [reply]
Thank you very much. Now it works.

1.  Import JSON File:
components/RegionsProvider/RegionsProvider.tsx
`import * as de from "../../locale/de-DE.json";`
2. Adding new case to importMessages:
components/RegionsProvider/RegionsProvider.tsx
 ```
export function importMessages(locale: string): LocaleMessages {
  switch (locale) {
    case "en-US":
      return sourceOfTruth;
    case "pl-PL":
      return pl;
    case "de-DE":
      return de;
    default:
      return sourceOfTruth;
  }
}
```
